packages/reports/addon/components/report-builder.hbs

```diff
@@ -1,38 +1,14 @@
 {{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 <div class="report-builder" ...attributes {{did-insert this.setupElement}}>
   {{#unless (is-mobile)}}
-    <div class="report-builder__side report-builder__side">
-      {{#if (or (gt this.allTables.length 1) (not this.hasValidLogicalTable))}}
-        <NaviTableSelect
-          @options={{this.allTables}}
-          @selected={{this.request.tableMetadata}}
-          @onChange={{update-report-action "UPDATE_TABLE"}}
-          @disabled={{@disabled}}
-        />
-      {{/if}}
-      <div class="report-builder__container--builder {{unless this.hasValidLogicalTable "report-builder__container--builder-skel"}}">
-        {{#if this.hasValidLogicalTable}}
-          <div class="report-builder__container report-builder__dimension-selector{{if @disabled " report-builder__container--disabled"}}">
-            <ColumnSelector
-              @title="Dimensions"
-              @allColumns={{append this.request.tableMetadata.dimensions this.request.tableMetadata.timeDimensions}}
-              @request={{this.request}}
-              @onAddColumn={{queue (fn (optional @onBeforeAddItem) this) (update-report-action "ADD_COLUMN_WITH_PARAMS")}}
-              @onAddFilter={{queue (update-report-action "ADD_DIMENSION_FILTER") this.onAddFilter}}
-            />
-        </div>
-        <div class="report-builder__container report-builder__metric-selector{{if @disabled " report-builder__container--disabled"}}" >
-          <ColumnSelector
-            @title="Metrics"
-            @allColumns={{this.request.tableMetadata.metrics}}
-            @request={{this.request}}
-            @onAddColumn={{queue (fn (optional @onBeforeAddItem) this) (update-report-action "ADD_COLUMN_WITH_PARAMS")}}
-            @onAddFilter={{queue (update-report-action "ADD_METRIC_FILTER") this.onAddFilter}}
-          />
-        </div>
-        {{/if}}
-      </div>
-    </div>
+    <ReportBuilder::Sidebar 
+      @disabled={{@disabled}}
+      @report={{@report}}
+      @lastAddedColumn={{@lastAddedColumn}}
+      @onBeforeAddItem={{fn (optional @onBeforeAddItem) this}}
+      @openFilters={{this.openFilters}}
```

Comment:
Could this be a generic hook like `onUpdate` and we use it to trigger the opening of the filter panel?

Reply:
I think this could probably take the same approach as https://github.com/yahoo/yavin/blob/c3748bf1d035864954b1a3bb7709b12646a87732/packages/reports/addon/consumers/request/column.ts#L45-L50
So that no matter where the filter is added from we always open the filter panel